src/openrct2/peep/Guest.cpp

```diff
@@ -3687,7 +3687,7 @@ void Guest::UpdateRideAdvanceThroughEntrance()
             actionZ += RideData5[ride->type].z;
         }
 
-        MoveTo((*loc).x, (*loc).y, actionZ);
+        MoveTo({ loc->x, loc->y, actionZ });
```

Comment:
Can't see on cellphone if loc is XY or XYZ. If first then it can be { *loc, actionZ }